plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -124,44 +150,197 @@ public function admin_styles() {
 			}
 		}
 
+		/**
+		 * Get the scripts used for registration.
+		 *
+		 * @return array
+		 */
+		private function get_scripts() {
+			$suffix  = Constants::is_true( 'SCRIPT_DEBUG' ) ? '' : '.min';
+			$version = Constants::get_constant( 'WC_VERSION' );
+
+			return array(
+				array(
+					'legacy_handle' => 'jquery-blockui',
```

Comment:
These legacy handles should have been prefixed to begin with, but unfortunately were not.  If we limit this to the admin, the chance of conflict is slightly lower but still exists.

There's obviously some duplication temporarily here until these are removed and in general registering these scripts early on every page will worsen performance across other pages.  But this is on average around `0.00028s` per page load that would have already been present on all WC pages.